metricflow/specs.py

```diff
@@ -315,7 +315,13 @@ def dimension_reference(self) -> DimensionReference:  # noqa: D
         return DimensionReference(element_name=self.element_name)
 
 
+# How to avoid circular import here
+from metricflow.model.objects.elements.measure import NonAdditiveDimensionParams
```

Comment:
Is there another way around this?

Reply:
Yup, it errors out with circular import if I put this to the top where the imports should be. Putting it where I put it in this PR works, but I didn't think it was a good idea to put it there unless it's actually okay.